src/pretix/presale/templates/pretixpresale/event/index.html

```diff
@@ -134,28 +134,28 @@ <h2 class="content-header">
                 {% if ev.location %}
                     <div class="info-row">
                         <span class="fa fa-map-marker fa-fw" aria-hidden="true"></span>
-                        <p><span class="sr-only">{% trans "Location" %}:</span>
+                        <p><span class="sr-only">{% trans "Where?" %}:</span>
                             {{ ev.location|linebreaksbr }}
                         </p>
                     </div>
                 {% endif %}
                 {% if ev.settings.show_dates_on_frontpage %}
                     <div class="info-row">
                         <span class="fa fa-clock-o fa-fw" aria-hidden="true"></span>
-                        <p>
-                            {{ ev.get_date_range_display }}
+                        <p><span class="sr-only">{% trans "When?" %}</span>
+                            <time datetime="{{ ev.date_from.isoformat }}">{{ ev.get_date_range_display }}</time>
                             {% if event.settings.show_times %}
                                 <br>
                                 <span data-time="{{ ev.date_from.isoformat }}" data-timezone="{{ request.event.timezone }}">
-                                    {% blocktrans trimmed with time=ev.date_from|date:"TIME_FORMAT" %}
-                                        Begin: {{ time }}
+                                    {% blocktrans trimmed with time=ev.date_from|date:"TIME_FORMAT" time_24=ev.date_from|time:"H:i" %}
+                                        Begin: <time datetime="{{ time_24 }}">{{ time }}</time>
```

Comment:
We should avoid bothering translators with HTML. Not at any cost, but whenever it's feasible. It only causes mistakes when translators get it wrong, and changing the markup breaks all translations.

The following could work (untested):

```
{% with ev.date_from|time:"H:i" as time_24h %}
    {% blocktrans trimmed with time="<time datetime='"|add:time_24|add:"'>"|add:time|add:"</time>"|safe %}
        Begin: {{ time }}
    {% endblocktrans %}
{% endwith %}

Same for the other fields of course

```